scripts/jenkins/cloud/ansible/roles/heat-generator/defaults/main.yml

```diff
@@ -21,6 +21,10 @@ rhel_image: "centos{{ rhel_os_version }}"
 # If SES is disabled, we need a greater disk size for swift
 disk_size: "{{ ses_enabled | ternary(2, 3) }}"
 
+openstack_bmc_image: "osbmc"
```

Comment:
Where does this image come from / how was it built?

Reply:
It is a opensuse image running the openstackbmc service [1]. I know that it would be better if the image was built using obs, however it would take long and probably not much value would be added as these VMs are just like "sidecar" simulating a BMC and not part of the product.

Reply:
Yes, that is ok.